README.md

```diff
@@ -5,14 +5,19 @@ Explore large, annoying graphs using hierarchies of dominating sets - because
 in space, no one can hear you miao!
 
 This is a collaboration between the
-[Theory In Practice](https://github.com/TheoryInPractice/) lab at NC
-State and the
-[Lab for Data Intensive Biology](https://github.com/dib-lab/) at UC
-Davis, generously supported by the Moore Foundation's
+[Theory In Practice](https://github.com/TheoryInPractice/) lab at University of Utah, the
+[Lab for Data Intensive Biology](https://github.com/dib-lab/) at UC Davis, and
+[Dr. Felix Reidl](https://www.dcs.bbk.ac.uk/about/people/academic-staff/felix/) at Birkbeck University of London,
+ generously supported by the Moore Foundation's
 [Data Driven Discovery Initiative](https://www.moore.org/initiative-strategy-detail?initiativeId=data-driven-discovery).
 
 ![spacegraphcats graph](https://github.com/spacegraphcats/spacegraphcats/raw/master/pics/logo.png)
 
+## Documentation
+
+This README file contains quickstart information.
+For use cases and other information, please see the spacegraphcats documentation at https://spacegraphcats.github.io/spacegraphcats.
```

Comment:
are we using readthedocs or github.io?

Reply:
should have https://spacegraphcats.github.io/ redirect, maybe.

Reply:
ooh that's a great idea...i don't know how to do that.

Reply:
voila https://github.com/spacegraphcats/spacegraphcats.github.io/blob/main/index.html